Altis_Life.Altis/core/functions/fn_keyHandler.sqf

```diff
@@ -284,11 +284,11 @@ switch (_code) do {
 
                     if (_locked isEqualTo 0) then {
                         _veh setVariable [format ["bis_disabled_Door_%1",_door],1,true];
-                        _veh animateSource [format ["Door_%1_source", _door], 0];
+                        _veh animate [format ["Door_%1_rot", _door], 0];
                         systemChat localize "STR_House_Door_Lock";
                     } else {
                         _veh setVariable [format ["bis_disabled_Door_%1",_door],0,true];
-                        _veh animateSource [format ["Door_%1_source", _door], 1];
```

Comment:
IIRC, we were having issues with doors across map (Tanoa was one - the big double doors). Does this work with them all?